tests/test_csv__provider.py

```diff
@@ -33,7 +33,7 @@
 from pygeoapi.provider.csv_ import CSVProvider
 
 
-path = 'pygeoapi-test.csv'
+path = '/tmp/pygeoapi-test.csv'
```

Comment:
I would expect to use a loading strategy as already implemented in other tests. An absolute path is only working in certain cases locally not in CI

Reply:
`Path='/tmp/pygeoapi-test.csv'` was the path already specified in test_csv_provider.py. For testing locally I changed it to `path='pygeoapi-test.csv'`. But now I reverted back the path location   to as it was before I.e.`Path='/tmp/pygeoapi-test.csv'`. It is same as master now. Do you want me to change the loading strategy? This would affect the master too..